test/unit/mock/profiles/unsupported_dependencies/inspec.yml

```diff
@@ -0,0 +1,13 @@
+name: unsupported_dependencies
+title: InSpec Profile
+maintainer: The Authors
+copyright: The Authors
+copyright_email: you@example.com
+license: Apache-2.0
+summary: An InSpec Compliance Profile
```

Comment:
Please give a purpose for this fixture profile.  We have many, and an understanding of what this is for will help your fellow developers.